inst/Upgrades.qml

```diff
@@ -0,0 +1,98 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:	"bayesianStateSpace"
+		fromVersion:	"0.1.0"
+		toVersion:		"0.16.4"
+
+		ChangeRename	{ from: "factors";										to: "fixedFactors"						}
+		ChangeRename	{ from: "dates";										to: "time"								}
+		ChangeRename	{ from: "postSummaryTable";								to: "posteriorSummaryTable"				}
+		ChangeRename	{ from: "posteriorSummaryCoefCredibleIntervalValue";	to: "posteriorSummaryCiLevel"			}
+		ChangeRename	{ from: "expectedModelSize";							to: "expectedPredictors"				}
+
+		ChangeRename	{ from: "checkboxAr";									to: "autoregressiveComponent"			}
+
+		ChangeJS
+		{
+			name: "lagSelectionMethod"
+			jsFunction: function(options)
+			{
+				switch(options["lagSelectionMethod"])
+				{
+					case "manualAR":		return "manual";
+					case "autoAR":			return "auto";
+				}
+			}
+		}
+
+		ChangeRename	{ from: "noLags";										to: "lags"								}
+		ChangeRename	{ from: "maxNoLags";									to: "maxLags"							}
+
+		ChangeRename	{ from: "checkboxLocalLevel";							to: "localLevelComponent"				}
+		ChangeRename	{ from: "checkboxLocalLinearTrend";						to: "localLinearTrendComponent"			}
+		ChangeRename	{ from: "checkboxSemiLocalLinearTrend";					to: "semiLocalLinearTrendComponent"		}
+
+		ChangeRename	{ from: "checkboxDynReg";								to: "dynamicRegregressionComponent"		}
+
+		ChangeRename	{ from: "DynRegLags";									to: "dynamicRegregressionLags"			}
+
+		ChangeJS
+		{
+			name: "seasonalities"
+			jsFunction: function(options)
+			{
+				options["seasonalities"] = options["seasonalities"].forEach(item=>{
+					item["number"]					= item["nSeason"];
+					item["duration"]				= item["seasonDuration"];
+
+					item["inverseGammaPriorSd"]		= item["sigma.guess"];
+					item["inverseGammaPriorN"]		= item["sample.size"];
+
+					item["normalPriorMean"]			= item["mu"];
+					item["normalPriorSd"]			= item["sigma"];
+					return item;
+				})
+			}
```

Comment:
This option also does not get upgraded, but I think there is actually a coding error as you are not returning anything from the `jsFunction`:
```suggestion
			jsFunction: function(options)
			{
				let seasonalities = options["seasonalities"].forEach(item=>{
					item["number"]					= item["nSeason"];
					item["duration"]				= item["seasonDuration"];

					item["inverseGammaPriorSd"]		= item["sigma.guess"];
					item["inverseGammaPriorN"]		= item["sample.size"];

					item["normalPriorMean"]			= item["mu"];
					item["normalPriorSd"]			= item["sigma"];
					return item;
				})
				
				return seasonalities;
			}
```